### PR TITLE
DDEX: packed updates

### PR DESCRIPTION
**Pull request by bargst**
Updates DDEX with latest API changes and add get_trades() and get_all_trades() call

### CODE REVIEW

File: pyexchange/ddex.py [hunk 1]
```diff
@@ -163,6 +207,56 @@ def cancel_order(self, order_id: str) -> bool:
...
+        assert(isinstance(page_number, int))
+
+        per_page = 100
+        page_filter = f"page={page_number}&per_page={per_page}"
+        result = self._http_get_signed(f"/v2/markets/{pair}/trades/mine?{page_filter}", {})['data']
+        totalPages = result['totalPages']
+        currentPage = result['currentPage']
+        self.logger.debug(f'totalPages={totalPages};currentPage={currentPage}')
+
+        # Oldest trades are on first page
```
Comment (reverendus) [inline pyexchange/ddex.py]: oldest or newest...?

File: pyexchange/ddex.py [hunk 2]
```diff
@@ -163,6 +207,56 @@ def cancel_order(self, order_id: str) -> bool:
...
+        assert(isinstance(page_number, int))
+
+        per_page = 100
+        page_filter = f"page={page_number}&per_page={per_page}"
+        result = self._http_get(f"/v2/markets/{pair}/trades?{page_filter}", {})['data']
+        totalPages = result['totalPages']
+        currentPage = result['currentPage']
+        self.logger.debug(f'totalPages={totalPages};currentPage={currentPage}')
+
+        # Oldest trades are on first page
```
Comment (reverendus) [inline pyexchange/ddex.py]: ?

File: pyexchange/ddex.py (hunk 1 above)
Comment (bargst) [reply pyexchange/ddex.py]: newest are now on top since the breakage, was an old comment will update !

File: pyexchange/ddex.py (hunk 2 above)
Comment (bargst) [reply pyexchange/ddex.py]: idem, now newest are first !

**Pull request merged**